templates/kubeflow/install.html

```diff
@@ -6,14 +6,16 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1KVyyvZsDyrfoi5tK2m5SLWs9YuHg9hgqVz0c8cWHsf8/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip--light is-deep is-bordered">
-  <div class="row">
-    <div class="col-8">
-      <h1>Install Kubeflow on Ubuntu</h1>
-      <p>Create and train machine learning models on your laptop, in your data center, or in the cloud.</p>
-    </div>
-    <div class="col-3 col-start-large-10 u-align--center u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/1ec1605a-kubeflow.svg?w=275" alt="" width="275" />
+<section class="p-strip is-deep is-bordered u-no-padding--top u-no-padding--bottom">
```

Comment:
You can add the p-strip--suru-topped to line 9 and remove p-strip u-no-padding--top u-no-padding--bottom. And also you can remove the next level div (line 10)

Reply:
Done